### ISSUE TITLE
provide simple way to seed data

### ISSUE THREAD

**Issue by kplimack** [yahoo/kafka-manager#347]
I would like to be able to spin up kafka-managers and provide seed data for the clusters instead of creating them manually.  Can you provide a way to seed data?  Backup/restore?

**Comment by jaymell** [comment]
I agree that this would be a very valuable feature. We would like to run Kafka Manager as a Docker service, but since there's not currently any way to configure the cluster without navigating through the GUI -- or submitting POST requests after startup -- we currently have to manually reconfigure the cluster every time a restart of the Docker service occurs.

At the very least, it would be helpful to be able to start Kafka Manager with relevant cluster data supplied by environment variables or a configuration file.
